src/languages/en.js

```diff
@@ -98,7 +98,7 @@ export default {
     hello: 'Hello',
     phoneCountryCode: '1',
     welcomeText: {
-        phrase1: 'Welcome to the New Expensify! Enter your phone number or email to continue.',
+        phrase1: 'Welcome to New Expensify! Enter your phone number or email to continue.',
```

Comment:
Based on David's [Vatican reference](https://expensify.slack.com/archives/C020EPP9B9A/p1626710044488300?thread_ts=1626126030.186700&cid=C020EPP9B9A), my understanding is the home page saying "the New Expensify" makes sense here.

Reply:
Cool, updated!